### PR TITLE
Add feedback section in the installation screen

### PR DESCRIPTION
**Pull request by KKoukiou**
In order to load the SVG image the build.js esbuild script had to be adjusted.

https://cockpit-logs.us-east-1.linodeobjects.com/pull-74-20231213-155623-1ca9887b-fedora-rawhide-boot/TestInstallationProgress-testBasic-installation-progress-complete-pixels.png

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
The PR is missing the actual jsx changes ;-) But I can sort of extract them from the force push diffs, and that LGTM.

Other than that, only bikeshedding - the aesthetic of the page was built on centered items, while this is left aligned. The "visual weight" of the qr code breaks the symmetry. But that's for those qualified with this kind of thing, I imagine...

File: src/components/installation/Feedback.jsx
Comment (VladimirSlavik) [inline src/components/installation/Feedback.jsx:38]: I think web "referer" is traditionally written with one r missing, although I'm not sure if that's still the case for this tech stack.

Comment (KKoukiou) [reply src/components/installation/Feedback.jsx:38]: No idea honestly  - seems 2 r is fine;

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer

**Review by KKoukiou — COMMENTED** [review]
@VladimirSlavik let's get the alignment / design final review from @Garrett

**Comment by garrett** [comment]
Is the screenshot the latest?

If so, it needs to be vertically aligned, with some space added. Right now, it's too scrunched up at the top and there's too much emphasis on the feedback part. It needs balance and spacing, like in the mockup.

(Additionally, we should really rebalance the progress page too.)

I guess the URL is a placeholder for the time being? It's way too long to be useful... nobody could remember _that_. (And like I've said multiple times, expecting someone to do everything that's needed to log in and/or create an account (including email, 2fa, etc.) workflow from within installation is very complex, so it's critical that the URL is memorable.) So I'm assuming this is a working placeholder for the time being, and _someone_ in the team is reaching out to someone in Fedora to make sure we can get a shorter, memorable URL... right? (Is someone already tasked with this? If not, they need to be. It's critical for this feature.)

But this is definitely a good start. We could merge it and fix the layout and links in followups.

**Comment by garrett** [comment]
I was able to run the commands in main, but in this PR, I get a traceback.

What I ran:

```bash
make deps
bots/image-download fedora-rawhide-boot
bots/image-download fedora-rawhide
make prepare-test-deps
make create-updates.img
./test/webui_testvm.py fedora-rawhide-boot
```

Traceback:

```py
❱ ./test/webui_testvm.py fedora-rawhide-boot
Serving HTTP on 0.0.0.0 port 8000 (http://0.0.0.0:8000/) ...
127.0.0.1 - - [04/Dec/2023 15:52:44] "GET / HTTP/1.1" 200 -
Serving HTTP on 0.0.0.0 port 8001 (http://0.0.0.0:8001/) ...
127.0.0.1 - - [04/Dec/2023 15:52:45] "GET / HTTP/1.1" 200 -
ERROR    Validating install media '/var/home/garrett/Project/anaconda/anaconda-webui/bots/images/fedora-rawhide-boot' failed: Must specify storage creation parameters for non-existent path '/var/home/garrett/Project/anaconda/anaconda-webui/bots/images/fedora-rawhide-boot'.
error: failed to get domain 'fedora-rawhide-boot-127.0.0.2-2201'
error: failed to get domain 'fedora-rawhide-boot-127.0.0.2-2201'
Traceback (most recent call last):
  File "/var/home/garrett/Project/anaconda/anaconda-webui/./test/webui_testvm.py", line 73, in <module>
    cmd_cli()
  File "/var/home/garrett/Project/anaconda/anaconda-webui/./test/webui_testvm.py", line 38, in cmd_cli
    machine.start()
  File "/var/home/garrett/Project/anaconda/anaconda-webui/test/machine_install.py", line 182, in start
    raise e
  File "/var/home/garrett/Project/anaconda/anaconda-webui/test/machine_install.py", line 167, in start
    Machine.wait_boot(self)
  File "/var/home/garrett/Project/anaconda/anaconda-webui/bots/machine/machine_core/ssh_connection.py", line 136, in wait_boot
    raise exceptions.Failure("Unable to reach machine {0} via ssh: {1}:{2}".format(
machine_core.exceptions.Failure: Unable to reach machine fedora-rawhide-boot-127.0.0.2-2201 via ssh: 127.0.0.2:2201
```

**Comment by garrett** [comment]
Summary of below: Grouping. Space _within_ the groups means we need even more space _between_ the groups.

---

Details that you can read if you're interested as to why:

My insistence on spacing comes from Gestalt Principles applied in UI design, specifically proximity.

(The Gestalt principles comes from the Bauhaus movement in Germany in 1919 - 1933 applying the recent-at-the-time topic of Gestalt psychology and it's one of the fundamental blocks of all types of design, including UI design. The artists back then basically tried to quantify the attributes of art and design. Here's a decent blog article on the subject: https://walterteng.com/gestalt-principles-ui)

The items are grouped together. However, since there are so many different elements, they really need extra emphasis on the grouping. In this case, that's the space, to separate the clusters of elements.

Without the additional space, all of it sort of blends together. The primary action, the reboot button, is lost in the middle. That space between the groups is fighting against the space between the elements in each group. As PatternFly already has a lot of space between the parts of the empty state pattern, we need to add _a lot more_ space to separate the feedback from the success+reboot area.

Note: There _are_ other ways to group, like drawing boxes around elements or using similar shapes or color. But those don't really apply here. The only other alternative that we could use is having a divider line to split up the groups further. (Which is "boxing" in a way, by having a 1-sided "box" and implying the rest of the borders around the rest... which makes use of the Gestalt concept of "closure".)

**Comment by rvykydal** [comment]
@garrett interesting and revealing explanation, thank you!
I'm going to review the PR after the layout is approved.

**Review by garrett — APPROVED** [review]
Thanks! Looks good!

**Review by adamkankovsky — APPROVED** [review]
Looks fine to me. Thanks

**Pull request merged**
